src/lib/components/blog/labels.svelte

```diff
@@ -34,15 +35,18 @@
     font-size: 1em;
 
     a {
-      background-color: var(--acm-light);
-      border-radius: 5px;
-      padding: 0.3em;
+      background-color: #eeeeee;
+      color: #212121;
+      border-radius: var(--size-sm);
+      border: 2px solid #e0e0e0;
+      padding: 0.25em 0.8em 0.25em 0.8em;
       text-decoration: none;
     }
 
     .selected {
-      background-color: var(--acm-gray);
-      color: var(--perma-light);
+      background-color: #81d4fa;
```

Comment:
how does using var(--acm-blue) and var(--acm-bluer) instead here look?

Reply:
All the other changes look pretty good, not sure if I'm rocking with this change though.
![Screenshot_20221206_011004](https://user-images.githubusercontent.com/111948487/206023739-958cd361-b446-4d37-ade4-b7cd5c02c5fa.png)